package/network/services/dropbear/patches/120-openwrt_options.patch

```diff
@@ -39,18 +39,20 @@
  
  /* Enable "Counter Mode" for ciphers. This is more secure than normal
   * CBC mode against certain attacks. It is recommended for security
-@@ -131,9 +131,9 @@ If you test it please contact the Dropbe
+@@ -131,10 +131,10 @@ If you test it please contact the Dropbe
   * If you disable MD5, Dropbear will fall back to SHA1 fingerprints,
   * which are not the standard form. */
  #define DROPBEAR_SHA1_HMAC
 -#define DROPBEAR_SHA1_96_HMAC
 +/*#define DROPBEAR_SHA1_96_HMAC*/
  #define DROPBEAR_SHA2_256_HMAC
 -#define DROPBEAR_SHA2_512_HMAC
+-#define DROPBEAR_MD5_HMAC
 +/*#define DROPBEAR_SHA2_512_HMAC*/
- #define DROPBEAR_MD5_HMAC
++/*#define DROPBEAR_MD5_HMAC*/
```

Comment:
This looks ok, OpenSSH also deactivated this by default in 7.2:
https://www.openssh.com/txt/release-7.2
I know of some people with some "old"? Microtik devices where you can not login via SSH from a recent Ubuntu any more, probably they depend on this, but I assume this is not a problem.

Reply:
Yes, this is also the reason why I think it is time that we can drop the MD5 HMAC support for dropbear as well.